lib/ts/recipe/webauthn/types.ts

```diff
@@ -492,12 +488,9 @@ export type RecipeInterface = {
               status: "OK";
               id: string;
               relyingPartyId: string;
-              relyingPartyName: string;
               origin: string;
-              userName: string;
-              userDisplayName: string;
+              email: string;
               timeout: string;
-              attestation: string;
               challenge: string;
```

Comment:
why did we remove relyingPartyName, userName, userDisplayName and attestation?

Reply:
I checked the schema and we don't store them in the generated options. `userName` is the email and i renamed it to be more simple.

Reply:
So ideally, the output of registerOptions function should be similar to the output of this function. For example, in the registerOptions output, we have:
```
user: {
                  id: string;
                  name: string; // user email
                  displayName: string; //user email
              };
```

Now in the output of getGeneratedOptions, it's totally different. So this is strange.

Reply:
Because we only store what is needed to verify the credentials. See the schema here (`webauthn_user_generated_options`) https://docs.google.com/document/d/1G7tO9_dSNi8wur3ajGg4pq-wiHatKDbHv2sBt-uSbQg/edit?tab=t.0#heading=h.td4zxgo1675k